src/ui/src/pages/auth/utils.ts

```diff
@@ -38,6 +39,12 @@ export const GetOAuthProvider = (): OAuthProviderClient => {
       return HydraClient;
     case 'oidc':
       return OIDCClient;
+    case undefined:
+      if (process.env.NODE_ENV === 'test' || process.env.JEST_WORKER_ID !== undefined) {
```

Comment:
Any way we can avoid test fixtures leaking to the main source code?

Reply:
Done.